From e2d1ef3fbca048d9d8e84898986e356ce126f4dd Mon Sep 17 00:00:00 2001
From: Jarek Jarcec Cecho <jarcec@apache.org>
Date: Tue, 23 Apr 2013 08:25:43 -0700
Subject: [PATCH 097/121] CLOUDERA-BUILD. Specifying correct avro classifier

CDH-11728: Hive pom files are depending on default avro classifier
---
 cloudera/maven-packaging/hive-exec/pom.xml  |    4 ++++
 cloudera/maven-packaging/hive-serde/pom.xml |    1 +
 2 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/src/cloudera/maven-packaging/hive-exec/pom.xml b/src/cloudera/maven-packaging/hive-exec/pom.xml
index 5546693..516a00d 100644
--- a/src/cloudera/maven-packaging/hive-exec/pom.xml
+++ b/src/cloudera/maven-packaging/hive-exec/pom.xml
@@ -163,6 +163,10 @@
           <groupId>org.apache.hive</groupId>
           <artifactId>hive-serde</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.avro</groupId>
+          <artifactId>avro-mapred</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
   </dependencies>
diff --git a/src/cloudera/maven-packaging/hive-serde/pom.xml b/src/cloudera/maven-packaging/hive-serde/pom.xml
index 53394aa..1a0b6fd 100644
--- a/src/cloudera/maven-packaging/hive-serde/pom.xml
+++ b/src/cloudera/maven-packaging/hive-serde/pom.xml
@@ -87,6 +87,7 @@
     <dependency>
       <groupId>org.apache.avro</groupId>
       <artifactId>avro-mapred</artifactId>
+      <classifier>hadoop2</classifier>
     </dependency>
     <dependency>
       <groupId>junit</groupId>
-- 
1.7.0.4

